packages/bot-web-ui/src/assets/sass/quick-strategy.scss

```
@@ -12,6 +15,18 @@
                 height: calc(100% - 200px);
                 max-height: 636px;
             }
+            .dc-tabs__content {
+                height: calc(100% - 4.8rem);
+            }
+        }
+    }
+    &-header {
+        &--#{$info-modal} {
+            border-bottom: none !important;
+            padding: 10px 8px 0px;
+        }
+        &__title--#{$info-modal} {
+            font-size: 16px !important;
```

```suggestion
            font-size: 16px !important;
```
          font-size: 1.6 rem !important;

---

Done, thank you! 🙏